store/search.js

```diff
@@ -1,27 +1,30 @@
 const FuzzyMatching = require("fuzzy-matching");
 
-const kWordBreakRegex = new RegExp(/\P{L}+/u);
```

Comment:
Why? `\P{L}` is unicode non-letter chars, do we want `a.b.c` to be itself or be `a`, `b`, and `c`?

Reply:
See: fffa60e2032e00d385af0816ce9e29276143b9dc 
```
chore: revert "feat(scrollback): add infinite message scrollback"

This reverts commit 862c1b0, reversing
changes made to 0bf5c20.
```
This commit made things too buggy, crashed a lot, and was hard to develop with.

Reply:
Sounds good, we could add that back eventually.